src/components/location-list/location-list.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {cityType} from '../../types';
+import {connect} from 'react-redux';
+import {ActionCreator} from '../../store/action';
+import LocationItem from '../location-item/location-item';
+
+const LocationList = (props) => {
+  const {locations, city, onLocationClick} = props;
+
+  return (
+    <React.Fragment>
+      <ul className="locations__list tabs__list">
+        {locations.map((location, id) => (
+          <LocationItem
+            key={id}
+            location={location}
+            activeLocation={city}
+            onLocationClick={onLocationClick}
+          />
+        ))}
+      </ul>
+    </React.Fragment>
```

Comment:
если возвращается одна нода, нет нужды оборачивать во фрагмент
```suggestion
      <ul className="locations__list tabs__list">
        {locations.map((location, id) => (
          <LocationItem
            key={id}
            location={location}
            activeLocation={city}
            onLocationClick={onLocationClick}
          />
        ))}
      </ul>
```